File: Algorithms/Python/0387/0387.py
 class Solution:
    def firstUniqChar(self, s: str) -> int:
        cnt = [0 for i in range(26)]
        for ch in s:
            cnt[ord(ch) - ord('a')] += 1
        for i in range(len(s)):
            if cnt[ord(s[i]) - ord('a')] == 1:
                return i
        return -1

File: Algorithms/C++/0387/0387.cpp
class Solution {
public:
    int firstUniqChar(string s) {
        vector<int> cnt(26);
        for (char ch : s) {
            ++cnt[ch - 'a'];
        }
        for (int i = 0; i < s.length(); i++) {
            if (cnt[s[i] - 'a'] == 1)  {
                return i;
            }
        }
        return -1;
    }
};
